dkg-gadget/src/worker.rs

```diff
@@ -782,13 +799,13 @@ where
 						return;
 					}
 				},
-				notification = self.block_import_notification.next().fuse() => {
-					if let Some(notification) = notification {
-						self.handle_import_notifications(notification);
-					} else {
-						return;
-					}
-				},
+				// notification = self.block_import_notification.next().fuse() => {
```

Comment:
Why is this commented out? Why not just remove it if its not being used, though shouldn't we be listening to finality notifications to start certain processes or no?